### PR TITLE
Fix/vm_savepoint_tests_mpi_failed

### PR DESCRIPTION
**Pull request by ajdas1**
## Purpose

Ran into some failed tests during `make savepoint_tests_mpi` on virtual machine. 

## Code changes:

- `stencils/pace/stencils/testing/test_translate.py`: fixed a TypeError that was popping up in line 365 (removed `[0]` from `case.grid[0].rank`)
- `fv3core/fv3core/testing/translate_dyncore.py`: test was failing as `ignore_near_zero_errors` was false for the `wsd` variable. Set `self.ignore_near_zero_errors = {"wsd": True}`

## Checklist
Before submitting this PR, please make sure:

- [x] You have followed the coding standards guidelines established at [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--BD7zigBMAhMZAPkeNENeuU2UAg-IlsYffZgTwyKEylty7NhY).
- [x] Docstrings and type hints are added to new and updated routines, as appropriate
- [x] All relevant documentation has been updated or added (e.g. README, CONTRIBUTING docs)
- [x] For each public change and fix in `pace-util`, HISTORY has been updated

Additionally, if this PR contains code authored by new contributors:

- [x] The names of all the new contributors have been added to CONTRIBUTORS.md

### CODE REVIEW

File: fv3core/fv3core/testing/translate_dyncore.py [hunk 1]
```diff
@@ -125,6 +125,7 @@ def __init__(
         # TODO: Fix edge_interpolate4 in d2a2c_vect to match closer and the
         # variables here should as well.
         self.max_error = 2e-6
+        self.ignore_near_zero_errors = {"wsd": True}
```
Comment (elynnwu) [inline fv3core/fv3core/testing/translate_dyncore.py:128]: Is there a reason this is not passed from the override file?

**Review by elynnwu — COMMENTED** [review]
Can you update the PR title to something more descriptive?

File: fv3core/fv3core/testing/translate_dyncore.py (hunk 1 above)
Comment (ajdas1) [reply fv3core/fv3core/testing/translate_dyncore.py:128]: Maybe @mcgibbon would know?

Comment (mcgibbon) [reply fv3core/fv3core/testing/translate_dyncore.py:128]: The units of ws (m/s) make it such that a 1e-18 difference is indeed very small, and shouldn't affect validation for any backend. My understanding is the override file is for backend-specific settings, not just as the default place to put validation constraints (though honestly, I'd feel better if all validation constraints were in yaml files and none were on translate classes like this).

**Review by elynnwu — APPROVED** [review]

**Comment by ajdas1** [comment]
already in main

**Pull request closed without merging**